ovis_bringup/launch/ovis_arm.launch

```diff
@@ -1,12 +1,12 @@
 <launch>
 
   <arg name="kinova_robotType" default="c2s6s000" />
-  <arg name="kinova_robotName" default="arm" />
+  <arg name="kinova_robotName" default="ovis" />
   <arg name="kinova_robotSerial" default="not_set" />
   <arg name="use_jaco_v1_fingers" default="false" />
   <arg name="feedback_publish_rate" default="0.1" />
 
-  <node name="$(arg kinova_robotName)" ns="ovis" pkg="kinova_driver" type="kinova_arm_driver" output="screen" cwd="node" args="$(arg kinova_robotType)">
+  <node name="$(arg kinova_robotName)" pkg="kinova_driver" type="kinova_arm_driver" output="screen" cwd="node" args="$(arg kinova_robotType)" respawn="true">
```

Comment:
Still hurts to look at. Could you add a task on Teams to document the need to fix this?

Reply:
I understand your concerns, but for the time being I think it's something we need to keep

Reply:
We agree on that. That's why I thought wise to create a Teams task to not let this eyesore be forgotten.

Reply:
- [ ] Not really an eyesore, here it's a patch but respawn is really nice to have on any mission essential node just in case it might crash

Reply:
Oh I misread your comment! I just added a task to address this issue :)

Reply:
> Let's not get pedantic, some autonomy standards enforce that applications crashing should equal end of mission in every possible scenario. In our context, we, however, do what we think is best for our use case. 
> 
> In the long term, that's a good way of missing problems, since detecting the node crashing might be complicated for users other than advanced ones.

Sure sounds like a great philosophical discussion to have over a beer at the pub 😉

I started investigation on the root issue without much success, seems like it is independant of our code (except launch and config files) i managed to reproduce it with the stock kinova_driver ros node, that seems to indicate me it might just be "la faute à élé" or network issues

Reply:
Ok, we'll look into it soon. In the meantime, I'll resolve this conversation so merging becomes available.